dist/cookie/{{cookiecutter.project_slug}}/screens/welcome/index.js

```diff
@@ -19,4 +25,41 @@ const WelcomeScreen = () => {
   )
 }
 
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: "#F8F8FC",
+    height: "100%"
+  },
+  scrollView: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "space-between",
+    padding: 20
+  },
+  group: {
+    alignItems: "center"
+  },
+  logo: {
+    height: 180,
+    width: 180,
+    backgroundColor: "#ECEEF8",
```

Comment:
This should be removed, it is introducing a weird background:
![image](https://github.com/crowdbotics/modules/assets/26097371/549612c7-b2de-4b60-bd2c-86a139b35355)

This is how it looks like with just the image:
![image](https://github.com/crowdbotics/modules/assets/26097371/d3a678e1-ce6c-4f79-a81f-c6d1db7e3ceb)